### PR TITLE
Dead code demolition: realtime package

### PR DESCRIPTION
**Pull request by raymondng**
> 1600 deletions (and more to come)
> Someone won a bet, I think.

### CODE REVIEW

**Comment by yingjieMiao** [comment]
Awesome.

**Comment by andrewconner** [comment]
@stkem I think Urban airship should go on Eliza. Thoughts? Sounds like Shoebox handling it is just adding latency. If so, does Eliza just call a method in `ShoeboxServiceClient` that relays it to `UrbanAirship.sendNotification`?

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala [hunk 1]
```diff
@@ -28,22 +27,21 @@ class UserController @Inject() (
...
-      Ok(Json.obj(
-        "token" -> device.token
-      ))
-    } getOrElse {
-      BadRequest(Json.obj(
-        "error" -> "Body must contain a token parameter"
-      ))
-    }
-  }
+//  def registerDevice(deviceType: String) = AuthenticatedJsonToJsonAction { implicit request =>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala]: I think we'll need to keep something like this (either in eliza or here). In general, we prefer to not comment out committed code. If it needs to come back later, we just use git to grab it,

Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala]: Agreed. That's the one place I wasn't sure about, so I didn't remove it outright. Looks like my suspicion was not unfounded -- will put it back in for now.

**Comment by andrewconner** [comment]
I think this one's safe. :+1:

**Comment by andrewconner** [comment]
Are we leaving UrbanAirship on shoebox for now?

**Comment by raymondng** [comment]
Thanks. Yes leaving it in shoebox for this pull request.

**Comment by stkem** [comment]
lgtm

**Pull request merged**